rust/worker/src/index/fulltext/types.rs

```diff
@@ -448,6 +448,10 @@ impl<'me> FullTextIndexReader<'me> {
             // Throw away the "value" since we store frequencies in the keys.
             token_frequencies.push((token.text.to_string(), res.1));
         }
+
+        if token_frequencies.len() == 0 {
```

Comment:
learning - this implied the tokens in the query don't match anything in index yeah?

Reply:
Actually, it is even more trivial than that. This will be hit when the tokenizer did not output any token for a query. The case that you are mentioning is handled in L#438 above